packages/backend/src/models/UserAttributesModel.ts

```diff
@@ -100,11 +101,21 @@ export class UserAttributesModel {
                 `${OrganizationMemberUserAttributesTable}.user_attribute_uuid`,
                 `${UserAttributesTable}.user_attribute_uuid`,
             )
+            .leftJoin(
+                GroupUserAttributesTable,
+                `${GroupUserAttributesTable}.user_attribute_uuid`,
+                `${UserAttributesTable}.user_attribute_uuid`,
+            )
             .leftJoin(
                 `users`,
                 `${OrganizationMemberUserAttributesTable}.user_id`,
                 `users.user_id`,
             )
+            .leftJoin(
+                `groups`,
+                `${GroupUserAttributesTable}.group_uuid`,
+                `groups.group_uuid`,
+            )
```

Comment:
We don't filter or select anything from `groups` so I don't think we need this join.

Reply:
Good call. I think I started by using it and got to a place where it's not needed.